openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Domains.spec.ts

```diff
@@ -582,7 +586,7 @@ test.describe('Domains Rbac', () => {
         const urlParams = new URLSearchParams(queryString);
         const qParam = urlParams.get('q');
 
-        expect(qParam).toContain(`domain.fullyQualifiedName:`);
+        expect(qParam).toContain('');
```

Comment:
Why we are asserting empty string?

Reply:
Because we now want All Domains to be visible for non admin users as well.

Reply:
Then it should be toEqual instead contain right?

Reply:
updated